privacyidea/api/validate.py

```diff
@@ -539,3 +540,56 @@ def trigger_challenge():
 
     return send_result(result_obj, details=details)
 
+
+@validate_blueprint.route('/polltransaction', methods=['GET'])
```

Comment:
To be more *RESTish* with GET-Requests we could add a second route-decorator:

@validate_blueprint.route('/polltransaction/<transacion_id>', methods=['GET'])

(compare GET requests in the token-endpoint)

Reply:
Hm yes, this actually seems more RESTish. Maybe we should *only* implement the ``/polltransaction/<transaction_id>`` endpoint?

Reply:
I've added a second endpoint in 250a3cf.